clar2wasm/src/wasm_generator.rs

```diff
@@ -110,6 +110,7 @@ pub enum GeneratorError {
     NotImplemented,
     InternalError(String),
     TypeError(String),
+    ArgumentLengthError(String),
```

Comment:
Could we have something easier to work with than a string here? A string forces us to work with regexes when we want to process the error.
We would need two `u32` for the expected and actual length. Then you can store in memory the 4 bytes of expected followed by the 4 bytes of actual.

Reply:
This error was no longer used, however the ones that are used are in `error_mapping.rs`.

They are the `ArgumentCountMismatch`, `ArgumentCountAtLeast` and `ArgumentCountAtMost` errors.